src/ui/server/controllers/insurance/policy/loss-payee-financial-details-uk/index.ts

```diff
@@ -107,6 +110,16 @@ export const post = async (req: Request, res: Response) => {
   }
 
   try {
+    const saveResponse = await mapAndSave.lossPayeeFinancialDetailsUk(payload, application);
+
+    if (!saveResponse) {
+      return res.redirect(PROBLEM_WITH_SERVICE);
```

Comment:
Do we need to add `console.error` for logging?

Reply:
we haven't anywhere else in the UI for this.  @ttbarnes

Reply:
No need to do this IMO, there is already logging and/or errors thrown in `mapAndSave` and the GQL call itself, so these would be logged.

If we add logging here I think it will just be adding noise.